### ISSUE TITLE
Steam controller only works with following combination: wired + BPM + no in-home streaming

### ISSUE THREAD

**Issue by JoranDox** [ValveSoftware/steam-for-linux#7055]
#### Your system information

* Steam client version (build number or date): I don't see where I can copy this, is it this? 
![image](https://user-images.githubusercontent.com/7152733/80132403-ea90ba80-859b-11ea-88f7-c7dd6a71509d.png)

* Distribution (e.g. Ubuntu): Ubuntu 20.04 rc
* Opted into Steam client beta?: [Yes/No] 
* Have you checked for system updates?: [Yes/No]

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

I put this as a support request and got redirected here so I'll just copypaste my post from there

Message from you on Apr 22 @ 5:05pm | 1 day and 2 hours ago
I recently built a new pc with linux (amd cpu & gpu, ubuntu 20.04) and now my steam controller only partly works (specifically tested with rocket league + proton 5) :
- out of game, it works fine
- in the steam overlay it works fine (bringing the steam overlay up mostly works
- in-game, nothing works, but I hear the sound of button inputs, so I'm assuming some config is being used but is transmissing inputs that are being ignored? even though the pop up states that it's using my bindings
- steam button + right bumper is accepted as a key sometimes, which leads me to believe that this is an issue with the controller configuration not applying the right config set?

at any rate I'm very confused, please advise what I can do to debug this better
I'm a programmer and accustomed to linux so if I need to do stuff in the terminal that's no problem

---

Message from you on Apr 22 @ 5:56pm | 1 day ago
I've tried a few more combinations of settings:

steam in big picture mode + steam controller wired + not streaming _works_!

if I don't turn on big picture mode (regardless of the in-game overlay), it doesn't work
if I try to use the steam controller wireless, it doesn't work
if I try to use in-home streaming, it doesn't work

also tested with monster hunter world in the meantime, I'm 99% sure it's not game dependant (these are the two major games in my library that I want to play with controller support)

I'd rather have the opposite where I have steam in regular (not big picture) mode (overlay can be big picture, I don't care) + use the steam controller wireless (because I can't reach my pc comfortably from the couch when wired, though I can fix this with a hub if there's no other way) + in-home streaming (so I can put my PC in the study and use my laptop as my client (ps streaming setup I tried was with a windows 10 laptop, if it matters I can try linux on there too))

Is this one issue or actually several different issues?

---
tl;dr: See title, if I play wireless, or with regular steam, or try to play using in-home streaming (regardless of whether I plug my steam controller in the host or client) it doesn't work.

**Comment by kisak-valve** [comment]
Hello @JoranDox, are you using the Valve-provided Steam package or the distro-provided Steam package? If you're using the distro-provided Steam package, please also install `steam-devices` via your package manager.

**Comment by JoranDox** [comment]
Hi @kisak-valve , thanks for the quick reply. I think I installed the distro-provided one but I'm not entirely sure anymore as I got the following when trying to install steam-devices:

> steam-devices is already the newest version (1:1.0.0.61-2ubuntu3).
> steam-devices set to manually installed.

**Comment by xsellier** [comment]
> Hello @JoranDox, are you using the Valve-provided Steam package or the distro-provided Steam package? If you're using the distro-provided Steam package, please also install `steam-devices` via your package manager.

I got the same issue, but installing `steam-devices` fixed it, thank you.

**Comment by JoranDox** [comment]
I also reinstalled `steam-devices` and apt updated & upgraded everything and it didn't change anything.

**Comment by JoranDox** [comment]
Can anyone tell me what the difference is between regular steam and big picture mode in regards to the controller working in-game? I'm thinking that difference covers both that and the in-home streaming issue (since I can't remotely enter big picture mode).

The wireless not working is likely a different issue.

**Comment by MCDELTAT** [comment]
I'm not sure If this is the same issue since the OP states it's a new Ubuntu 20.04 install but I was just able to resolve a similar issue. 

I upgraded from Linux Mint 19.3 (Ubuntu 18.04) to Linux Mint 20 (Ubuntu 20.04) and had previously in the place the UDEV rules recommended by Value to get the Steam Controller to work. As stated, I was able to use my controller in all of the menus but not in any games. I installed the packages with `sudo apt install steam steam-devices` with no change. I moved the UDEV rules out and rebooted and everything worked. So the regular Ubuntu Packages should be sufficient alone to get it working.

**Comment by JoranDox** [comment]
I've since moved from gnome to kde plasma, and about 90% of my issues with just about anything disappeared. I haven't tried in-home streaming anymore but steam controller seems to work fine with or without BPM for the few games I've tried it with.
I've also played guacamelee via remote play together (friend was host, I was joining, both had controllers) and that worked fine.

I'll see if I can run the same tests that I did a few months ago and report back, and then see what those UDEV rules do to the tests too (& thanks @MCDELTAT for the input)

**Comment by kisak-valve** [comment]
Hello @JoranDox, are you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.
